doc/plugins/determine_mime_type.md

```diff
@@ -28,8 +30,33 @@ The following analyzers are accepted:
 | `:mini_mime`    | Uses the [mini_mime] gem to determine the MIME type from the file extension. Note that unlike other solutions, this analyzer is not guaranteed to return the actual MIME type of the file.                                                                                        |
 | `:content_type` | Retrieves the value of the `#content_type` attribute of the IO object. Note that this value normally comes from the "Content-Type" request header, so it's not guaranteed to hold the actual MIME type of the file.                                                               |
 
-A single analyzer is not going to properly recognize all types of files, so you
-can build your own custom analyzer for your requirements, where you can combine
+You'll need to ensure the file utility or gem is in installed on your system
+for Shrine to use the analyzer.
+
+## Analyzer Options
+
+Currently, Marcel is the only analyzer that accepts options to be passed in.
+Analyzer options can be set in one of two ways:
+
+```rb
+plugin :determine_mime_type, analyzer: :marcel, options: { filename_fallback: true }
```

Comment:
I think it would be more explicit if `:options` was renamed to `:analyzer_options`. What do you think?